src/components/DatePicker.tsx

```diff
@@ -0,0 +1,168 @@
+import { rgba } from 'polished'
+import Picker from 'react-date-picker'
+import styled, { css } from 'styled-components'
+
+const today = new Date()
+const minDate = new Date(today.getFullYear(), today.getMonth(), today.getDate())
```

Comment:
This might be outdate when user using Kyberswap in the end of date. Cost of constructing `Date` is quite cheap. So just put it inside component rendering is fine.

Reply:
This is just a suggestion. Another way to make minDate `new Date(Math.floor(Date.now() / 1000 / 86400) * 1000 * 86400)`.

Reply:
I get @hungdoansy idea, I also used in charts, I have shorter version 
new Date(Date.now() - Date.now() % 86400000)

Reply:
2 different Date.now() might return different value @XiaoYhun

Reply:
maybe not :))))) 
![Screenshot_1](https://user-images.githubusercontent.com/9592090/206618346-ef5f07eb-db97-4ba3-a8eb-b291f4c43c70.png)

Reply:
How could you compare your Macbook Pro 16inch M1 Pro 16/512gb with user possible with old intel i3. On their machine it might be slow and return different value 😆

Reply:
anyway if this data not required to be exactly correct value to be `...00000`, this solution is acceptable.

Reply:
appreciate your ideas, guys, but I don't want to change =))

Reply:
This is not improvement suggestion. This is about bug potential. You might need to explain why this bug never be happended or have to change it.

Reply:
what's wrong with my code, I just want to calc minDate = today at 0h 0m 0s
minDate = new Date(today.getFullYear(), today.getMonth(), today.getDate())

Reply:
hahaha, its ok to use it like that, but IMO it just look ugly.
another version to achieve it:  new Date( new Date().setHours(0, 0, 0, 0) )

Reply:
It's because of `minDate` is 1 times calculated. Consider this case: `minDate` is calculated at the end of date. Then the place using this `minDate` value (component `DatePicker`) rendered in the next day. So `DatePicker` has use outdated value of `minDate`. So I suggest that dont 1 times calculated `minDate` but move it to the rendering phase of `DatePicker` and recalculating everytime of that component renders.

Reply:
agree, I moved it inside component.